src/sentry/conf/server.py

```diff
@@ -1596,3 +1598,6 @@ def get_sentry_sdk_config():
         'topic': KAFKA_EVENTS,
     },
 }
+
+# Symbolicator service
+SENTRY_SYMBOLICATOR_URL = "http://localhost:42069"
```

Comment:
Is this also going to be added to `sentry devservices`?

Reply:
We intend to have a config value that influences whether it's used (and downloaded as devservice), but generally yes.